virocon/plotting.py

```diff
@@ -154,7 +254,29 @@ def plot_2D_isodensity(model, sample, model_desc=None, swap_axis=False, ax=None)
     return ax
 
 
+
 def plot_2D_contour(contour, sample=None, design_conditions=None, model_desc=None, swap_axis=False, ax=None):
+    """
+    Beschreibung 
+       
+    Parameters
+    ----------
+    contour: Contour
+        The environmental contour that should be plotted.
+    sample : ndarray of floats, optional
+        The environmental data sample that should be plotted against the fit.
```

Comment:
A 2D data sample that should be plotted along the contour. Shape: (number of realizations, 2)